avocado/core/__init__.py

```diff
@@ -188,6 +188,14 @@ def register_core_options():
                          key_type=list,
                          help_msg=help_msg)
 
+    help_msg = ('By default Avocado runners will use the {uri} of a test as '
+                'its identifier. Use a custom f-string identifer in order to '
+                'change it.')
+    stgs.register_option(section='runner',
+                         key='identifier_fmt',
```

Comment:
IMO we shouldn't be saving a few characters in something that is user visible.  I strongly suggest (I'm begging, really :smile: ) that you use `identifier_format` instead.